### PR TITLE
Update sensor.imap.markdown

### PR DESCRIPTION
**Pull request by noxhirsch**
Update IMAP Sensor configuration variable

### CODE REVIEW

File: source/_components/sensor.imap.markdown [hunk 1]
```diff
@@ -30,11 +30,29 @@ sensor:
...
-- **name** (*Optional*): Name of the IMAP sensor.
-- **username** (*Required*): Username for the IMAP server.
-- **password** (*Required*): Password for the IMAP server.
-- **folder** (*Optional*): The IMAP folder to watch.
+{% configuration %}
+server:
+  description: The IP address or hostname of the IMAP server.
+  required: true
+  type: string
+port:
```
Comment (klaasnicolaas) [inline source/_components/sensor.imap.markdown:38]: It has a default port: 993

File: source/_components/sensor.imap.markdown [hunk 2]
```diff
@@ -30,11 +30,29 @@ sensor:
...
+  type: string
+username:
+  description: Username for the IMAP server.
+  required: true
+  type: string
+password:
+  description: Password for the IMAP server.
+  required: true
+  type: string
+folder:
```
Comment (klaasnicolaas) [inline source/_components/sensor.imap.markdown:55]: default: inbox

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]

File: source/_components/sensor.imap.markdown (hunk 1 above)
Comment (noxhirsch) [reply source/_components/sensor.imap.markdown:38]: Thank you! Added 👍

File: source/_components/sensor.imap.markdown (hunk 2 above)
Comment (noxhirsch) [reply source/_components/sensor.imap.markdown:55]: Thank you! Added 👍

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @noxhirsch!

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**